lua/astrocommunity/lsp/lsp-signature-nvim/README.md

```diff
@@ -0,0 +1,5 @@
+# lsp_signature.nvim
+
+**Repository:** <https://github.com/ray-x/lsp_signature.nvim>
+
+Show function signature when you type.
```

Comment:
Could you add some info here about hint_enable and why it's disabled?

Reply:
done :)